packages/manager/tools/component-rollup-config/src/index.ts

```diff
@@ -52,6 +61,19 @@ const generateConfig = (opts, pluginsOpts) =>
         sass({
           insert: true,
           output: false,
+          options: {
+            importer(url) {
```

Comment:
in order to handle sass imports correctly in rollup i had to manually resolve url using node resolver
(see: https://rollupjs.org/guide/en/#why-isnt-node-resolve-a-built-in-feature)